lib/ZcapClient.js

```diff
@@ -0,0 +1,228 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+import {DEFAULT_HEADERS, httpClient} from '@digitalbazaar/http-client';
+import {SECURITY_CONTEXT_V2_URL} from 'jsonld-signatures';
+import {CapabilityDelegation} from 'ocapld';
+import {generateId} from 'bnid';
+import jsigs from 'jsonld-signatures';
+import {signCapabilityInvocation} from 'http-signature-zcap-invoke';
+const {Ed25519Signature2018} = jsigs.suites;
+
+export class ZcapClient {
+  /**
+   * An object that can make HTTPS requests such as node's https.Agent or ky.
```

Comment:
```suggestion
   * An object that manages connection persistence and reuse for HTTPS requests.
```

Reply:
This has been fixed... Github not allowing me to commit two suggestions for the same source line.